azext_iot/tests/central/__init__.py

```diff
@@ -31,6 +31,7 @@
     "azext_iot_central_dns_suffix",
     "azext_iot_teststorageaccount",
     "azext_iot_central_storage_container",
+    "azext_iot_teststoragecontainer",
```

Comment:
I see that you added this test variable but you do not use it anywhere aside from the delete. (like you do not check if it has a value and use that value, etc). I strongly recommend not pulling in this variable/ having delete depend on this variable so that we can keep the iot hub and central containers separate (and if the user wants to have the same container they would have to set both variables to point to the same container).

Reply:
Sure. Let me remove this. Just found that this variable also work for iot central int tests so I add it. It should be my misunderstanding.